extension/manifest.json

```diff
@@ -806,6 +806,16 @@
 			"js": ["scripts/global/functions/drag-drop-touch.js", "scripts/content/crimes/ttCrimes.js", "scripts/features/quick-crimes/ttQuickCrimes.js"],
 			"run_at": "document_end"
 		},
+		{
+			"matches": ["https://www.torn.com/loader.php?sid=crimes*"],

```

Comment:
No longer the correct page, it moved to page.php.

Reply:
Crimes 2.0 loads for me on `loader.php` as well, but I changed as Torn's link in sidebar takes to `page.php`.